open_spiel/integration_tests/playthroughs/python_liars_poker.txt

```diff
@@ -0,0 +1,820 @@
+                                             ◯◯
+                                             ◯◯
+                                             ◯◯
+                                             ◯◯
+                                             ◯◯
+                                             ◯◯
+                                             ◯◯
+                                             ◯◯
+InformationStateTensor(1).player: ◯◉
+InformationStateTensor(1).private_hand = [3, 3, 3]
+InformationStateTensor(1).rebid_state: ◯
+InformationStateTensor(1).counts_state: ◯
+InformationStateTensor(1).bid_history: ◯◯
+                                       ◯◯
+                                       ◯◯
+                                       ◯◯
+                                       ◯◯
+                                       ◯◯
+                                       ◯◯
+                                       ◯◯
+                                       ◯◯
+                                       ◯◯
+                                       ◯◯
+                                       ◯◯
+                                       ◯◯
+                                       ◉◯
+                                       ◯◯
+                                       ◯◯
+                                       ◯◯
+                                       ◯◯
+InformationStateTensor(1).challenge_history: ◯◯
+                                             ◯◯
+                                             ◯◯
+                                             ◯◯
+                                             ◯◯
+                                             ◯◯
+                                             ◯◯
+                                             ◯◯
+                                             ◯◯
+                                             ◯◯
+                                             ◯◯
+                                             ◯◯
+                                             ◯◯
+                                             ◯◯
+                                             ◯◯
+                                             ◯◯
+                                             ◯◯
+                                             ◯◯
+ObservationString(0) = "p0 hand:[1, 1, 2] rebid:[0] counts:[0]"
+ObservationString(1) = "p1 hand:[3, 3, 3] rebid:[0] counts:[0]"
+PublicObservationString() = "p0 rebid:[0] counts:[0]"
+PrivateObservationString(0) = "p0 hand:[1, 1, 2]"
+PrivateObservationString(1) = "p1 hand:[3, 3, 3]"
+ObservationTensor(0).player: ◉◯
+ObservationTensor(0).private_hand = [1, 1, 2]
+ObservationTensor(0).rebid_state: ◯
+ObservationTensor(0).counts_state: ◯
+ObservationTensor(1).player: ◯◉
+ObservationTensor(1).private_hand = [3, 3, 3]
+ObservationTensor(1).rebid_state: ◯
+ObservationTensor(1).counts_state: ◯
+Rewards() = [0, 0]
+Returns() = [0, 0]
+LegalActions() = [1, 16, 17, 18, 19]
+StringLegalActions() = ["Challenge", "Bet", "Bet", "Bet", "Bet"]
```

Comment:
Different action id's should have different strings, so probably an amount /count needs to be added to Bet